101-sig-image-version-create/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,216 @@
+      "metadata": {
+        "description": "The description of this Shared Image Gallery resource."
+      }
+    },
+    "location": {
+      "type": "string",
+      "defaultValue": "[resourceGroup().location]",
+      "metadata": {
+        "description": "Location of the Shared Image Gallery."
+      }
+    },
+    "subnetName": {
+      "type": "string",
+      "defaultValue": "mySubnet",
+      "metadata": {
+        "description": "Subnet Name."
+      }
+    },
+    "virtualNetworkName": {
+      "type": "string",
+      "defaultValue": "virtualNetwork",
+      "metadata": {
+        "description": "Virtual Network Name."
+      }
+    },
+    "networkInterfaceName": {
+      "type": "string",
+      "defaultValue": "nic01",
+      "metadata": {
+        "description": "Network Interface Name."
+      }
+    },
+    "addressPrefixes": {
+      "type": "string",
+      "defaultValue": "172.16.0.0/16"
+    },
+    "subnets": {
+      "type": "string",
+      "defaultValue": "172.16.0.0/24"
+    },
+    "virtualMachineName": {
+      "type": "string",
+      "defaultValue": "vm001"
+    },
+    "virtualMachineSize": {
+      "type": "string",
+      "defaultValue": "Standard_D2s_v3"
+    },
+    "osDiskType": {
+      "type": "string",
+      "defaultValue": "Premium_LRS"
+    },
+    "adminUsername": {
+      "type": "string",
+      "defaultValue": "azureuser"
+    },
+    "adminPassword": {
+      "type": "securestring"
+    }
+  },
+  "variables": {
+    "galleryName": "[uniqueString(resourceGroup().id)]",
+    "imageName": "[concat(variables('galleryName'), '/', parameters('galleryImageDefinitionName'))]",
+    "vnetId": "[resourceId('Microsoft.Network/virtualNetworks', parameters('virtualNetworkName'))]",
+    "subnetRef": "[concat(variables('vnetId'), '/subnets/', parameters('subnetName'))]"
+  },
+  "resources": [
+    {
+      "type": "Microsoft.Compute/galleries",
+      "name": "[variables('galleryName')]",
+      "apiVersion": "2019-12-01",
+      "location": "[parameters('location')]",
+      "properties": {
+        "description": "[parameters('description')]"
+      }
+    },
+    {
+      "type": "Microsoft.Compute/galleries/images",
+      "name": "[variables('imageName')]",
+      "apiVersion": "2019-12-01",
+      "location": "[parameters('location')]",
+      "dependsOn": [
+        "[resourceId('Microsoft.Compute/galleries', variables('galleryName'))]"
+      ],
+      "properties": {
+        "description": "Sample Gallery Image Description",
+        "osType": "Linux",
+        "osState": "Specialized",
+        "endOfLifeDate": "2030-01-01",
+        "identifier": {
+          "publisher": "myPublisher",
+          "offer": "myOffer",
+          "sku": "mySku"
+        },
+        "recommended": {
+          "vCPUs": {
+            "min": "1",
+            "max": "64"
+          },
+          "memory": {
+            "min": "2048",
+            "max": "307720"
+          }
+        }
+      }
+    },
+    {
```

Comment:
remove all the unused resources (everything but the gallery and image)

Reply:
What resources are not being used?